### PR TITLE
Adds password checker with the appropriate colour and messages

### PR DESCRIPTION
**Pull request by coderaashir**

### CODE REVIEW

**Comment by AkankshaBodhankar** [comment]
Squash your commits.

File: javascripts/passchecker.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    }
+    var z = 1, x = a + b + c;
+     if(z > x){
+        pt.text("Password Should have Special Characters, Numbers and at-least one Capital Letter"); 
+        return x; 
+    }
+    if(x == 1){
+        pt.text("Weak"); 
+    }
+    if(x==2){
```
Comment (AkankshaBodhankar) [inline javascripts/passchecker.js]: Spaces after each == or =

File: registration.php [hunk 2]
```diff
@@ -48,6 +58,20 @@
   <script type="text/javascript" src="javascripts/validation-v12.js"></script>
   <script src="javascripts/sweetalert.min.js"></script>
   <script src="javascripts/sweetalert.js"></script>
+  <style> 
+  #pass-text{
+    padding-top: 25px; padding-left: 10px; 
+  }
```
Comment (AkankshaBodhankar) [inline registration.php]: Add style in Css file only

**Comment by coderaashir** [comment]
I've made the changes, but how do you squash commits with multiple files changed? (i haven't been using the command line git till now)

**Comment by AkankshaBodhankar** [comment]
If you haven't that's not a problem. What you can do is make the above changes and then a new PR. Are you familiar with Github Desktop. You can download it it's easy gui interface.
Take back up of your changes
Then, fork the project and view it in github desktop. You will have an option to view in github desktop near the download Zip option 
After that you will have a local copy of project in computer, make the changes there and then a new PR for the changes made using Github Desktop :-) 
You will find many tutorials online. Even if you aren't able to squash after that we will still consider your task as your code looks good.

**Pull request closed without merging**